src/store/preferences.ts

```diff
@@ -47,9 +47,9 @@ export const usePreferencesStore = defineStore('preferences', {
 
       // NB: 18446744073709551615 is the max u64 number, which is the required type for `max_time_before_invoice.secs`
       // Number should never need to exceed this amount.
-      if (maxTimeBeforeInvoice > 18446744073709551615) {
-        maxTimeBeforeInvoice = 18446744073709551615
-      } 
+      if (maxTimeBeforeInvoice > 18446744073709550000) {
+        maxTimeBeforeInvoice = 18446744073709550000
+      }
```

Comment:
That looks great and very simple but I have problems understanding what is this change for :-)

Reply:
Ok, so you are correcting a max int here. Thanks!